src/index.ts

```diff
@@ -464,7 +464,7 @@ prog
     const logger = await createProgressEstimator();
     if (opts.format.includes('cjs')) {
       try {
-        await util.promisify(mkdirp)(resolveApp('./dist'));
+        await ensureDistFolder();
```

Comment:
I just realized there's an `ensureDistFolder()` call literally 4 lines before this... So this call is actually redundant (and therefore the whole try-catch too); not sure why it was needed for #130 .

Also, using `outputFile` instead of `writeFile` in `writeCjsEntryFile` (like I did in #367 ) can replace the need for `ensureDistFolder`

Reply:
Followed up with this change in #406